smart-contracts/contracts/cl-vault/src/test_tube/initialize.rs

```diff
@@ -216,8 +243,8 @@ pub mod initialize {
         // Create Alice account
         let alice = app
             .init_account(&[
-                Coin::new(1_000_000_000_000, DENOM_BASE),
-                Coin::new(1_000_000_000_000, DENOM_QUOTE),
+                Coin::new(1_000_000_000_000, "uatom"),
```

Comment:
NIT: to maintain consistency shouldnt this be `DENOM_BASE` and `DENOM_QUOTE` in whole snippet here

Reply:
This is just a test case. Actually in the whole initialize.rs file DENOM_BASE and DENOM_QUOTE have been deprecated because we will pass them from outside, and the initialize functions will take them as argument.

Reply:
okay got it